Player.cpp

```diff
@@ -0,0 +1,37 @@
+#include "Player.h"
+#include "TextureHolder.h"
+
+using namespace sf;
+
+Player::Player()
+{
+	m_points = 0;
+	m_health = 20;
+	m_dead = false;
+
+	
+}
+
+Player::~Player()
+{
+
+}
+
+
+
+
+
+bool Player::isAlive()
+{
+	if (m_health <= 0)
+	{
+		m_dead = true;
+	}
+	else
+	{
+		m_dead = false;

```

Comment:
Since this variable is never really "read" it could actually be removed!
The `isAlive` function could simply `return m_health > 0`!